end-to-end/package-jazzer-js.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/sh
+
+cd ..
+npm install
+npm run build
+npm run build --workspace='@jazzer.js/fuzzer'
+
+sed_version_and_mv() {
```

Comment:
What is the reason for stripping the version? node can move the generated *.tgz files to where you need them like this:
`npm pack --workspaces --pack-destination end-to-end`.
And you can remove the following from `package.json`, because the correct file names will be added when you run `npm install --save-dev *.tgz`:
```
"@jazzer.js/bug-detectors": "file:jazzer.js-bug-detectors.tgz",
"@jazzer.js/core": "file:jazzer.js-core.tgz",
"@jazzer.js/fuzzer": "file:jazzer.js-fuzzer.tgz",
"@jazzer.js/hooking": "file:jazzer.js-hooking.tgz",
"@jazzer.js/instrumentor": "file:jazzer.js-instrumentor.tgz",
"@jazzer.js/jest-runner": "file:jazzer.js-jest-runner.tgz",
```

Reply:
Ah, yes, that makes sense!

Reply:
I like the idea but putting them into a temp dir would result in `package.json` changing to `"@jazzer.js/bug-detectors": "file:/var/folders/...tgz",` on every run instead.

Reply:
I mean to copy everything that is in `end-to-end` directory into a tmp directory before starting the test and to treat that as the test directory. The original package.json will never be changed.

Reply:
That would be simpler but I think it would make local testing more awkward.